### PR TITLE
Kind Deployment Simple Test and README Update

### PR DESCRIPTION
**Pull request by phudtran**
This PR does the following

- Adds Kind deployment and  guide for Merak + Alcor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,22 +108,86 @@ The deployment settings such as container image and replicas can be changed by e
...
+
+![Successful Merak Deployment](docs/images/merak_successful_deployment.jpg)
+
+#### Step 2: Run test
+
+You can use the prebuilt test tool as shown below.
 
 ```
 ./tools/teste2e/bin/teste2e
 ```
```
Comment (xieus) [inline README.md:175]: Don't forget to mention 10 :-)

Comment (phudtran) [reply README.md:175]: I've fixed this!

Comment (xieus) [reply README.md:175]: oh, did you push to the PR? 

https://github.com/phudtran/merak/tree/deployment-optimize#step-2-run-test still shows no parameter passing.

Comment (phudtran) [reply README.md:175]: Yes the binary has been updated. Users will not need to pass in command line arguments

Comment (xieus) [reply README.md:175]: aha, that was the trick. can we keep the binary while allowing user to pass in the number of vhost to be started, which is more configurable.

**Comment by xieus** [comment]
One more general comment, maybe do it in next PR. 

One more session for test reset, e.g., allow user to delete vhost, delete services, scale out to more instances. This can be a separate Readme, not the main readme.

File: README.md (hunk 1 above)
Comment (phudtran) [reply README.md:175]: I think we can think about expanding this teste2e tool in a future release (Configure VPC, Subnet, sg, vhost evms, etc. For now I'd like to keep it as is with no config from the user as changing the number of vhosts also affects the way the evms are scheduled.

Comment (xieus) [reply README.md:175]: Okay sounds good to me.

**Review by xieus — APPROVED** [review]

**Pull request merged**